dashboard/db/schema.rb

```diff
@@ -1485,6 +1485,7 @@
     t.integer "source_course_offering_id"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.boolean "receives_marketing"
```

Comment:
Do we want to have a default value and a null value set for this too? (like [here](https://github.com/code-dot-org/code-dot-org/pull/54758/files#diff-bf90ed5047a62239db22703a2b939c6c3ad3c9aecd5859601afd4aa9cfddbb60R1426))

Reply:
Oooo yes yes good call. Rolling back, updating and re-running now!